tests/serverpod_test_module/serverpod_test_module_server/config/passwords.yaml

```diff
@@ -0,0 +1,8 @@
+shared:
+  authKeySalt: 'salty'
+  AWSAccessKeyId: 'AKIAXR2IQI6JRDCVBT3A'
+  AWSSecretKey: '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'
```

Comment:
Question: Are the AWS keys required for the module or could they be removed?

Reply:
Good point, they can be removed.

Reply:
848825c